recipes/xgboost/meta.yaml

```
@@ -94,6 +104,9 @@ outputs:
       string: cuda_{{ cuda_major }}_py{{ py_version }}_{{ build_number }}
       ignore_run_exports:
         - nvcc
+      ignore_run_exports_from:
+        - nvcc_linux-64 # [linux64]
+        - nvcc_linux-aarch64 # [aarch64]
```

This probably can be simplified like so. Though this can be handled later

```suggestion
      ignore_run_exports_from:
        - {{ compiler('nvcc') }}
```

---

Agreed. Let's do that on branch-22.06 in a follow up PR.